api/server.py

```diff
@@ -199,24 +243,24 @@ def decorated(*args, **kwargs):
                 raise AuthError(
                     {"code": "token_expired", "description": "token is expired"}, 401
                 )
-            except jwt.JWTClaimsError:
+            except jwt.JWTClaimsError as e:
                 raise AuthError(
                     {
                         "code": "invalid_claims",
-                        "description": "incorrect claims,"
-                        " please check the audience and issuer",
+                        "description": "incorrect claims, please check the audience and issuer",
                     },
                     401,
                 )
-            except Exception:
+            except Exception as e:
                 raise AuthError(
                     {
                         "code": "invalid_header",
-                        "description": "Unable to parse authentication" " token.",
+                        "description": "Unable to parse authentication token.",
```

Comment:
what do you think about also logging what `e` is? In case it has useful details for debugging.

Reply:
I guess I am wondering if there could be an Exception that isnt necessarily an AuthError, or related to the token

Reply:
Totally - that's a great suggestion to include the exception in the exception's error message! b475a90ba028d0f232438fabc017f1cedba2b475